src/main/java/org/spongepowered/api/variant/VariantQueryTypes.java

```diff
@@ -22,24 +22,20 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.data.type;
+package org.spongepowered.api.variant;
 
+import org.spongepowered.api.data.type.DyeColor;
+import org.spongepowered.api.data.type.Material;
 import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
 
-public final class LogAxes {
-
-    private LogAxes() {}
+@SuppressWarnings("unchecked")
+public final class VariantQueryTypes {
 
     // SORTFIELDS:ON
 
-    public static final LogAxis NONE = DummyObjectProvider.createFor(LogAxis.class, "NONE");
-
-    public static final LogAxis X = DummyObjectProvider.createFor(LogAxis.class, "X");
+    public static final VariantQueryType<Material> MATERIAL = DummyObjectProvider.createFor(VariantQueryType.class, "MATERIAL");
```

Comment:
With my comment above about Variety vs Material, this is an example where "Variants" will win overall as you can use different varieties to describe something unique (almost mergeable with properties).